build.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
```

Comment:
Let's ship this with another one PR.
- We should use build.sh in Dockerfile to avoid doubling. 
- Need to update Linux build instructions

Reply:
I agree, just didn't move it separately yet :)

Reply:
Removed from this PR